api/src/opentrons/protocol_runner/json_translator.py

```diff
@@ -9,6 +9,7 @@
     DeckSlotLocation,
     PipetteName,
 )
+from opentrons.protocol_engine.types import Liquid
```

Comment:
Outside of the `opentrons.protocol_engine` module, try to only import from the top-level

```suggestion
from opentrons.protocol_engine import Liquid
```